### DISCUSSION TITLE
part-ii/2020-126/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#66] [Question comments]
Part II 2020 Quantum Information and Computation: Paper 3, Section I, $10 C$ 

https://questions.tripos.org/part-ii/2020-126/

**Comment by thenewtonium** [comment]
Part (iv) is badly worded; apparently what they mean is show how you can prepare the state $\ket{g}$ from the state $\frac{1}{2}(\ket{g}+\ket{b_1}+\ket{b_2}+\ket{b_3})$ using these operators.
